npm run build fails if --strictNullChecks is set to true

If I set `--strictNullChecks` to true in tsconfig.json, `npm run build` fails : 

```
$ npm run build

> schemats@1.0.1 build C:\node-projects\schemats
> tsc

test/test.ts(49,9): error TS2345: Argument of type 'undefined' is not assignable to parameter of type 'string | null'.

```

The reason in that in the following line, we pass namespace as undefined :
https://github.com/SweetIQ/schemats/blob/master/test/test.ts#L49

while in https://github.com/SweetIQ/schemats/blob/master/src/index.ts#L37, only string is allowed for namespace parameter.

PR is coming.

--- a/src/index.ts
+++ b/src/index.ts
@@ -34,7 +34,7 @@ export function getTime() {
     return `${yyyy}-${MM}-${dd} ${hh}:${mm}:${ss}`
 }
 
-export async function typescriptOfSchema(db: Database, namespace: string, tables: string[], schema: string|null = 'public',
+export async function typescriptOfSchema(db: Database, namespace: string|null, tables: string[], schema: string|null = 'public',
                                          commandRan: string, time: string): Promise<string> {
     if (namespace) {
         console.warn('[DEPRECATED] Generation schema with namespace is deprecated.')

--- a/test/test.ts
+++ b/test/test.ts
@@ -46,7 +46,7 @@ async function testGeneratingTables(db: Database) {
     let outputFile = (process.env.CIRCLE_ARTIFACTS || './test/artifacts') + '/osm.ts'
     let formattedOutput = await typescriptOfSchema(
         db,
-        undefined,
+        null,
         ['users'],
         null,
         extractCommand(

